[debug] [2020-04-08T16:22:29.861Z] ----------------------------------------------------------------------
[debug] [2020-04-08T16:22:29.863Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\user\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js init
[debug] [2020-04-08T16:22:29.864Z] CLI Version:   8.0.2
[debug] [2020-04-08T16:22:29.864Z] Platform:      win32
[debug] [2020-04-08T16:22:29.864Z] Node Version:  v13.0.1
[debug] [2020-04-08T16:22:29.865Z] Time:          Wed Apr 08 2020 21:22:29 GMT+0500 (Узбекистан, стандартное время)
[debug] [2020-04-08T16:22:29.865Z] ----------------------------------------------------------------------
[debug] [2020-04-08T16:22:29.865Z] 
[debug] [2020-04-08T16:22:29.876Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-08T16:22:29.905Z] Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.
    at GoogleAuth.getApplicationDefaultAsync (C:\Users\user\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:160:19)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async GoogleAuth.getClient (C:\Users\user\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:502:17)
    at async GoogleAuth.getAccessToken (C:\Users\user\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:524:24)
[error] 
[error] Error: An unexpected error has occurred.
